src/core/proto/dst_entry_tcp.cpp

```diff
@@ -222,7 +222,9 @@ ssize_t dst_entry_tcp::fast_send(const iovec *p_iov, const ssize_t sz_iov, xlio_
             m_sge[i].addr = (uintptr_t)p_tcp_iov[i].iovec.iov_base;
             m_sge[i].length = p_tcp_iov[i].iovec.iov_len;
             if (is_zerocopy) {
-                if (PBUF_DESC_MKEY == p_tcp_iov[i].p_desc->lwip_pbuf.pbuf.desc.attr) {
+                if (PBUF_DESC_EXPRESS == p_tcp_iov[i].p_desc->lwip_pbuf.pbuf.desc.attr) {
```

Comment:
Can we refactor to more readable
`auto desc_attr = p_tcp_iov[i].p_desc->lwip_pbuf.pbuf.desc.attr`
if (PBUF_DESC_EXPRESS == desc_attr || PBUF_DESC_MKEY == desc_attr)
 ....
else if (PBUF_DESC_MDESC == desc_attr || ...)
  ....